tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -101,6 +101,60 @@ spec:
               topologyKey: kubernetes.io/hostname
             weight: 1
       containers:
+      - args:
+        - --single-child
+        - --
+        - /usr/bin/tail
+        - -n+1
+        - -F
+        - /var/log/placement/placement-api.log
+        command:
+        - /usr/bin/dumb-init
+        env:
+        - name: CONFIG_HASH
+          value: n67chfch95hc9h67ch5dbh5cch97h5c9h5d8h66bh89h54hd6h669h555hcch566hb4h4h64bh4hf7hc9hfdh5cfhb8h5fh55h5f5h5cfhbbq
+        - name: KOLLA_CONFIG_STRATEGY
+          value: COPY_ALWAYS
+        image: quay.io/podified-antelope-centos9/openstack-placement-api@sha256:9bcfea1d6ff9ad44243a744cf9e4228e19a83a8eade1161de7fb5fd848befd5d
```

Comment:
I would rather ditch the mentioning of the image hash as it will change independently of the operator code